### PR TITLE
Feat/condo/doma 5429/organization news core model

### PR DESCRIPTION
**Pull request by AleX83Xpert**
New model for organization's news.
This is the core-only model. Additional fields will be added in the future.

### CODE REVIEW

File: apps/condo/domains/organization/schema/OrganizationNewsItem.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        },
+
+        type: {
+            schemaDoc: 'The news item type',
+            type: Select,
+            options: NEWS_TYPES,
+            isRequired: true,
+        },
+
+        receiversCriteria: {
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/OrganizationNewsItem.js:47]: please don't use a JSON field! or provide the field schema

File: apps/condo/domains/organization/schema/OrganizationNewsItem.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+const { Json } = require('@open-condo/keystone/fields')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/organization/access/OrganizationNewsItem')
+const { NEWS_TYPES } = require('@condo/domains/organization/constants')
+
+
+const OrganizationNewsItem = new GQLListSchema('OrganizationNewsItem', {
```
Comment (pahaz) [inline apps/condo/domains/organization/schema/OrganizationNewsItem.js:15]: probably we need a new `news` domain

File: apps/condo/domains/organization/schema/OrganizationNewsItem.js (hunk 1 above)
Comment (AleX83Xpert) [reply apps/condo/domains/organization/schema/OrganizationNewsItem.js:47]: Yes, validation will be created after I'll understand the format.

File: apps/condo/domains/organization/schema/OrganizationNewsItem.js (hunk 2 above)
Comment (AleX83Xpert) [reply apps/condo/domains/organization/schema/OrganizationNewsItem.js:15]: Or a miniapp, as discussed a bit earlier. 🤔

Comment (AleX83Xpert) [reply apps/condo/domains/organization/schema/OrganizationNewsItem.js]: Decided to move into the new domain.

File: apps/condo/domains/news/schema/OrganizationNewsItem.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        title: {
+            schemaDoc: 'The news item title',
+            type: Text,
+            isRequired: true,
+        },
+
+        body: {
+            schemaDoc: 'The news item main body',
+            type: Text,
```
Comment (nomerdvadcatpyat) [inline apps/condo/domains/news/schema/OrganizationNewsItem.js:34]: Is that always just a text?
No iamges/markdown?

File: apps/condo/domains/news/schema/OrganizationNewsItem.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/news/access/OrganizationNewsItem')
+const { NEWS_TYPES } = require('@condo/domains/news/constants')
+
+const OrganizationNewsItem = new GQLListSchema('OrganizationNewsItem', {
+    schemaDoc: 'The news item created by the organization to show on resident\'s mobile devices',
+    fields: {
+
```
Comment (akarjakin) [inline apps/condo/domains/news/schema/OrganizationNewsItem.js:16]: As I understant, targets, publication date, etc. will be added later ?

Comment (AleX83Xpert) [reply apps/condo/domains/news/schema/OrganizationNewsItem.js:16]: Yes, right

File: apps/condo/domains/news/schema/OrganizationNewsItem.js (hunk 3 above)
Comment (AleX83Xpert) [reply apps/condo/domains/news/schema/OrganizationNewsItem.js:34]: For now, this is a text, yes.

**Review by akarjakin — APPROVED** [review]

**Review by nomerdvadcatpyat — APPROVED** [review]

**Pull request merged**